lib/mongoose_push/metrics/telemetry_metrics.ex

```diff
@@ -7,6 +7,10 @@ defmodule MongoosePush.Metrics.TelemetryMetrics do
     TelemetryMetricsPrometheus.Core.child_spec(metrics: metrics())
   end
 
+  def pooler do
+    [{:telemetry_poller, measurements: periodic_measurements(), period: 5_000}]
```

Comment:
Would it make sense to allow configuring the `period`? Prometheus by default scraps metrics every minute so perhaps we don't need to calculate these stats every 5s.

Reply:
Will have it changed to 30 seconds.